backend/src-common/src/main/java/com/siemens/sw360/components/summary/ProjectSummary.java

```diff
@@ -54,4 +57,32 @@ protected static void setSummaryFields(Project document, Project copy) {
             }
         }
     }
+
+    protected static void setDefaultFields(Project document, Project copy) {
+        copyField(document, copy, _Fields.ID);
```

Comment:
As ID, NAME, and VERSION are always set and probably must always be set, I'd put them into an extra method like `setMinimalFiields`